bld/namelist_files/use_cases/2000_cam6_noresm.xml

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0"?>
```

Comment:
Are the differences between this file and the NorESM2 release version necessary?

I think we should also restore the other CAM-Nor use case files:
```
bld/namelist_files/use_cases/2000_cam6_noclb_oslo.xml
bld/namelist_files/use_cases/2000_cam6_noclb_oslonudge.xml
bld/namelist_files/use_cases/2000_cam6_noresm_frc2.xml
bld/namelist_files/use_cases/2000_cam6_oslo.xml
bld/namelist_files/use_cases/2000_cam6_oslonudge.xml
```